src/components/MissionView.js

```diff
@@ -1,6 +1,66 @@
-const MissionView = () => (
-  <>
-    <p>Hello this is missions</p>
-  </>
-);
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchMissions, changeStatus } from '../redux/slices/missionReducer';
+
+function MissionView() {
+  const missions = useSelector((state) => state.missions);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchMissions);
+    // eslint-disable-next-line
+  }, []);
```

Comment:
Instead of disabling the linters how about adding the dependency parameter of dispatch in the useEffect(() => {...}, [dispatch])